drivers/watchdog/wdt_gecko.c

```diff
@@ -246,11 +246,6 @@ static int wdt_gecko_init(const struct device *dev)
 {
 	const struct wdt_gecko_cfg *config = dev->config;
 
-#ifdef CONFIG_WDT_DISABLE_AT_BOOT
-	/* Ignore any errors */
-	wdt_gecko_disable(dev);
-#endif
-
```

Comment:
What's the reason for moving this further down?

Reply:
this seems to be some kind of leftover after splitting the work into smaller chunks for PRs. Fixed